src/plugins/wizard/public/application/components/top_nav.tsx

```diff
@@ -25,10 +40,21 @@ export const TopNav = () => {
     },
   } = services;
   const rootState = useTypedSelector((state) => state);
+  const dispatch = useTypedDispatch();
   const hasUnappliedChanges = useTypedSelector(
```

Comment:
Can we rename this to something more meaningful since we now also have a `hasChange` value. Something like `hasDraftAgg`

Reply:
Agree, hasDraftAgg does make a lot more sense~